### DISCUSSION TITLE
Unsplitting a panel does not seem to work.

### DISCUSSION THREAD

**Discussion by kusrinivasan** [Eugeny/tabby#5410] [Q&A]
I am on MacOS 20.6.0 using Tabby.sh Version 1.0.169 (1.0.169) in  "Things you can do with Tabby #5259" it says using "Command + Shift" one can move the pane to another location or move it away as "Tab". I can't seem to do the latter that is moving it back as a Tab. I need this badly as that is how I can multi-task by making something primary and moving another one in, and after the work is done replace that pane with other windows. 

Thanks in advance for any pointers.

**Comment by Eugeny** [comment]
Fixed in the nightly.
